aws/resource_aws_waf_web_acl.go

```diff
@@ -212,6 +218,17 @@ func resourceAwsWafWebAclRead(d *schema.ResourceData, meta interface{}) error {
 	}
 	d.Set("name", resp.WebACL.Name)
 	d.Set("metric_name", resp.WebACL.MetricName)
+
+	tagList, err := conn.ListTagsForResource(&waf.ListTagsForResourceInput{
+		ResourceARN: aws.String(arn),
+	})
+	if err != nil {
+		return fmt.Errorf("Failed to get WAF ACL parameter tags for %s: %s", d.Get("name"), err)
+	}
+	if err := d.Set("tags", tagsToMapWAF(tagList.TagInfoForResource.TagList)); err != nil {
```

Comment:
This functionality is available in the `keyvaluetags` library as:

```suggestion
	if err := d.Set("tags", keyvaluetags.WafKeyValueTags(tagList.TagInfoForResource.TagList).IgnoreAws().Map()); err != nil {
```